rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -68,12 +65,12 @@ namespace rocal{
         return ptr;
     }
 
-    py::object wrapper_copy_to_output(RocalContext context, py::array_t<unsigned char> array)
+    py::object wrapper(RocalContext context, py::array_t<unsigned char> array)
     {
```

Comment:
Parenthesis style needs to be changed

Reply:
complete files coding style is diff,can be issued as a seperate PR